src/services/character/character.service.ts

```diff
@@ -16,7 +16,7 @@ export class CharacterService {
   public async getEntity(id: string):Promise<Document | null> {
     return await characters.findOne({ ownerId: id }, (err: Error, characterFound) => {        
       if (characterFound) return characterFound
-      else throw err
+      else return console.log(`Error: Character not found because: ${err}`)
```

Comment:
I never that someone return a console.log before.
I do not even know what is the return type and I am not sure that the log will be sent

Reply:
Oh... Sorry. It was because the throw was breaking the bot. I don't need to stop the process of character creation if the character wasn't found. Typescript doesn't scream and the console.log executes itself anyway.

Reply:
Good to know but still this is not standard.
Throwing errors is good.
Catching them when you think that having error should be ok is better.
What you need to do here is simply catch the error to avoid the break of the app.

Reply:
Ok. I see! I will do that in the next pull request!